modules/core/04-channel/keeper/grpc_query.go

```diff
@@ -5,6 +5,7 @@ import (
 	"strconv"
 	"strings"
 
+	errorsmod "cosmossdk.io/errors"
```

Comment:
I think v7 is still using sdkerrors

Reply:
I believe errorsmod is only on main, see https://github.com/cosmos/ibc-go/commit/a24a6c65f14ceee5e9c8dd3f0f79d63439f1db23